packages/toolpad-core/src/contexts/NavigationContext.ts

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+/**
+ * @ignore - internal component.
+ */
+
+export interface NavigationPageItem {
+  kind?: 'page';
+  title?: string;
```

Comment:
Are we intentionally making the `title` optional?
This title is being used for the list item titles (required prop in `ListItemText`, and also as part of the unique ids used for navigation items. But for the ids we could maybe use the item index instead - not sure why it's using the title but it was probably just the first approach.

Reply:
we can fall back to the segment.

Reply:
Ok, I thought about that. We can do that fallback + I have to try it out but it should be possible to just remove the title from those ids.